qiime/core/archiver.py

```diff
@@ -130,39 +175,32 @@ def provenance(self):
     def orphan(self, pid):
         self._pid = pid
 
-    def load_data(self, loader):
-        if not os.path.exists(self._data_dir):
-            self._extract_data()
+    def get_index_paths(self):
```

Comment:
No changes were made this method except to move it into a better location and remove the guards checking if the archive had been extracted.